lib/ruby-mws/api/feed.rb

```diff
@@ -111,6 +111,7 @@ def content_for_product_list_inventory(opts={})
               xml.Inventory {
                 xml.SKU entry[:isbn]
                 xml.Quantity entry[:quantity]
+                xml.FulfillmentLatency entry[:fulfillment_latency]
```

Comment:
will anything bad happen if entry[:fulfillment_latency] is nil? it wouldn't set it to less than the 2-day default amount, will it?

Reply:
I will update it to default to 5 (|| 5 ).
Blurby is the only client, and I updated it to send this, so I had thought it wouldn't matter.